examples/armada.yaml

```diff
@@ -20,7 +20,7 @@ armada:
         source:
           type: git
           location: git://github.com/att-comdev/openstack-helm
-          subpath: common
+          subpath: helm-toolkit
```

Comment:
@gardlt Some time has passed since this feedback, you can feel free to merge this in if you like and we can just handle the wider common rename as some subsequent effort?